custom/icds_reports/sqldata/exports/dashboard_usage.py

```diff
@@ -13,7 +13,7 @@
 
 class DashBoardUsage:
 
-    title = 'Dashboard Activity'
```

Comment:
@challabeehyv  I think this is also used as the tab name in the excel sheet can you check with QA/Saket whether that also needs to changed to this or not. ?

Reply:
QA wanted consistency in the name everywhere. So changed it accordingly. Let me confirm with QA team

Reply:
@Rohit25negi Confirmed this as a requirement from QA.

Reply:
you mean title of the tab in excel report also needs to be changed along with csv file name?

Reply:
@Rohit25negi latest : file names should have 'Report' suffix.

Reply:
> @challabeehyv I think this is also used as the tab name in the excel sheet can you check with QA/Saket whether that also needs to changed to this or not. ?

we are setting the tab name separately

Reply:
> you mean title of the tab in excel report also needs to be changed along with csv file name?

in short, excel tab should have Dashboard Activity and csv file should have Dashboard Activity Report

Reply:
I see